charts/stable/recyclarr/docs/installation_notes.md

```diff
@@ -0,0 +1,15 @@
+# Installation Notes
+
+- Shell into the chart via the recyclarr cardview in Apps -> 3 dot -> shell.
+- Set the `commands` to `/bin/bash`.
+- Run the command below to init the recyclarr's files in `/config`.
+
+```terminal
+recyclarr create-config
+```
```

Comment:
If that's the only thing needed to do in shell, add a `installContainer` that executes that. 
(`installContainers` run only on the first install of the app)